docs/user/debugging/python.md

```diff
@@ -17,15 +17,15 @@ If the `GLEAN_LOG_PINGS` environment variable is set to `true`, pings are
 logged to the console on `DEBUG` level whenever they are submitted.
 
 Make sure that when you configure logging in your application, you set the
-level for the `glean` logger to `DEBUG` or higher. Otherwise pings won't be
+level for the Python logger to `DEBUG` or higher. Otherwise pings won't be
 logged even if `GLEAN_LOG_PINGS` is set to `true`.
 
-You can set the logging level for Glean to `DEBUG` as follows:
+You can set the logging level for Python to `DEBUG` as follows:
 
 ```python
 import logging
 
-logging.getLogger("glean").setLevel(logging.DEBUG)
+logging.basicConfig(level=logging.DEBUG)
```

Comment:
as above, this configures a basic logger (log to stdout) and changes the level to debug, whereas the old line only changed the glean logger itself and left the rest of the config to the application.

So maybe we need a combination here?
A "quick, give me everything to stdout" and a "here's how you configure only the glean part, the rest is up to you"